elements/pfe-card/src/pfe-card.scss

```diff
@@ -4,40 +4,13 @@ $LOCAL: card;
 
 
 $variables: (
-
   // Individual padding overrides available
   PaddingTop: calc(#{pfe-var(container-spacer)} * 2),
```

Comment:
@starryeyez024 These are the only variables that _need_ to be set on the host.  By putting the other variables inline with their property calls, we are making it possible to set the value for those variables at any level in the cascade (not just on the component).  So now you could do: `:root { --pfe-card--spacing: 20px; }` and have it influence all the cards on that page.

Reply:
Before we could only do: `pfe-card { --pfe-card--spacing: 20px; }`

Reply:
Your logic makes total sense (and seems like a really important pivotal point on how we ought to do variables across the board), however I see `spacing` in the pfe-card.scss variables map (and thus is printed in `:host`) so I can't override it at a global level. 
This `:root { --pfe-card--BorderRadius: 100px; }` on the other hand,  does work as you describe. Do you want to remove the `spacing` var too?

![image](https://user-images.githubusercontent.com/456863/79153215-7edc6f80-7d9b-11ea-994a-1862da3e5cd1.png)

Reply:
To your point, I almost wonder if we should adjust how the mixin `pfe-print-variables` and the function `pfe-local` both work. Could the `pfe-print-variables` print sass variables instead, that the `pfe-local` function could utilize?

Coming at it from the other direction, let me re-ask a question this way:  is the general goal to
1. have a map at the top where a dev can either specify custom values, or global references for ease, like this:
```scss
$variables: (
  Color: orange, //custom
  Padding: pfe-var(container-spacer) //global
);
```
2. in the remainder of the sass file, your values are always the pfe-local function, like this:
```css
:host {
  Color: #{pfe-local(Color)};
}
```

3. the resulting CSS always prints empty local vars first, and then the values from the variables map?
```css
:host {
  color: var(pfe-card--Color,  orange ); 
                     ^empty                   ^custom
  padding: var(pfe-card--Padding, var(--pfe-theme--container-spacer, 16px));
                     ^empty                               ^global
}
```

Let me know if you want to chat/screenshare & discuss more!

Reply:
I love it!  I just updated the branch to implement this idea!